src/_cffi_src/openssl/ssl.py

```diff
@@ -331,6 +332,14 @@
 const SSL_METHOD *DTLSv1_server_method(void);
 const SSL_METHOD *DTLSv1_client_method(void);
 
+/* Added in 1.0.2 */
+const SSL_METHOD *DTLSv1_2_method(void);
+const SSL_METHOD *DTLSv1_2_server_method(void);
+const SSL_METHOD *DTLSv1_2_client_method(void);
```

Comment:
Can we skip the explicitly versioned ones and just do `DTLS_{server,client,}_method`? That's the prefered API + using `OP_NO` to disable unwanted protocols